templates/kubernetes/what-is-kubernetes.html

```diff
@@ -538,10 +672,10 @@ <h4 class="p-card__title">
         <hr>
         <div class="p-card__content">
           <p>
-            Deploy Canonical's Charmed Kubernetes, a highly available, pure upstream, multi-node Kubernetes cluster. It's a fully automated, model-driven approach to Kubernetes that takes care of logging, monitoring and alerting and provides application lifecycle automation capabilities.
+            Deploy Canonical's Charmed Kubernetes, a highly available, pure upstream, multi-node Kubernetes cluster. It's a fully automated, model-driven approach to Kubernetes that takes care of logging, monitoring and alerting , while also providing application lifecycle automation capabilities.
```

Comment:
I think it's better without a space after "alerting" even though theres is in the copy doc. 
```suggestion
            Deploy Canonical's Charmed Kubernetes, a highly available, pure upstream, multi-node Kubernetes cluster. It's a fully automated, model-driven approach to Kubernetes that takes care of logging, monitoring and alerting, while also providing application lifecycle automation capabilities.
```